docs/skopeo-sync.1.md

```diff
@@ -66,6 +66,8 @@ Print usage statement.
 
 **--scoped** Prefix images with the source image path, so that multiple images with the same name can be stored at _destination_.
 
+**--append-suffix** _tag-suffix_ String to append to destination tags
```

Comment:
```suggestion
**--append-suffix** _tag-suffix_ String to append to destination tags.
```
It would be nice to have a small example with a sample command and then an expected result in the examples section.

Reply:
Sure - example and missing `.` added